README.md

```diff
@@ -0,0 +1,95 @@
+# Outlier Detect
+
+**Author**: Ben Birnbaum (benjamin.birnbaum@gmail.com)
+
+**Contributors**: Faisal Lalani (flalani@dimagi.com), Brian DeRenzi (bderenzi@dimagi.com)
+
+This is a Python implemention of the Multinomial Model Algorithm (MMA) for outlier detection in survey responses. It works by comparing a set of answers by a particular interviewer to the set of answers by fellow responders. Based on how similar this observed distribution is to the expected, an outlier score is given to the interviewer & question pair.
+
+For a deeper understanding of the functionality, refer to the following papers:
+
+B. Birnbaum, B. DeRenzi, A. D. Flaxman, and N. Lesh.  Automated quality control
+for mobile data collection. In DEV ’12, pages 1:1–1:10, 2012.
+
+B. Birnbaum. Algorithmic approaches to detecting interviewer fabrication in
+surveys.  Ph.D. Dissertation, Univeristy of Washington, Department of Computer
+Science and Engineering, 2012.
+
+(See http://bbirnbaum.com/publications/ for PDF versions of
+these papers.)
+
+## Data Required
+
+The primary input for the algorithm is a csv file containing survey responses, formatted as follows:
+
+| interviewer_id  | question_a | question_b |
+| -------------   | ------------- | ------------- |
+| 1  | answer_1a  | answer_1b  |
+| 2  | answer_2a  | answer_2b  |
+| 3  | answer_3a  | answer_3b  |
+
+The distribution of answers from Interviewer 1 are compared to the distribution of answers from both Interviewers 2 & 3. Likewise for Interviewer 2 with the answers of 1 & 3 and for Interviewer 3 with 1 & 2.
+
+## Usage
+
+1. Once you've downloaded or cloned this repository, you need to first make sure the necessary libararies are installed on your machine. You can do this easily by running the following commands:
+
+```bash
+python3 setup.py install
```

Comment:
Just checking -- why do they need to run this? Are we trying to install the algorithm as a library?

Reply:
I left it from Ben's original documentation (see his README) in case the setup included some component required that I missed in the requirements.txt.

Reply:
Right. I don't think this is the recommended way to use the library right now. This will install the `outlierdetect` module using Distutils. See, e.g.: https://stackoverflow.com/questions/1471994/what-is-setup-py

Additionally, it will use Ben's version and contact info as per the `setup.py` file. I think we should mention that the library is available to be installed globally--and perhaps we should use the pip recommendation. I also think we should update the `setup.py` file to bump the version and put Dimagi's contact info

Reply:
Updated to reflect your feedback and suggestions!